extensions/google-chrome/src/hooks/useHistorySearch.tsx

```diff
@@ -8,6 +8,11 @@ const whereClauses = (tableTitle: string, terms: string[]) => {
   return terms.map((t) => `(${tableTitle}.title LIKE '%${t}%' OR ${tableTitle}.url LIKE '%${t}%')`).join(" AND ");
 };
 
+/**
+ * Generates a query to search the history for a list of terms.
+ *
+ * Using `last_visit_time > 0` to filter out bookmarks.
+ */
```

Comment:
Appreciate the comment.

I'm unfamiliar with this part of the code. Chrome stores the bookmarks in history? I thought it had a separate bookmarks file.